geoips/commandline/geoips_run.py

```diff
@@ -3,36 +3,89 @@
 Runs the appropriate script based on the args provided.
 """
 
-from glob import glob
-from importlib import resources
-from os.path import basename
-from subprocess import call
+from geoips.commandline.args import add_args
+from geoips.commandline.run_procflow import main
+from geoips.commandline.geoips_command import GeoipsCommand, GeoipsExecutableCommand
+from data_fusion.commandline.args import add_args as data_fusion_add_args
```

Comment:
This assumes that data_fusion is installed and raises an error if it is not.

```python
> geoips -h
Traceback (most recent call last):
  File "/home/jsolbrig/anaconda3/envs/geoips-github/bin/geoips", line 5, in <module>
    from geoips.commandline.commandline_interface import main
  File "/local/home/jsolbrig/geoips/github/geoips/geoips/commandline/commandline_interface.py", line 19, in <module>
    from geoips.commandline.geoips_run import GeoipsRun
  File "/local/home/jsolbrig/geoips/github/geoips/geoips/commandline/geoips_run.py", line 9, in <module>
    from data_fusion.commandline.args import add_args as data_fusion_add_args
ModuleNotFoundError: No module named 'data_fusion'
```

Reply:
This is making me wonder whether the CLI should, itself, accept plugins. I think that after we finish the order-based procflow, though, it won't be necessary.

Reply:
What would you like me to do about this one? Do we want to do an optional import, move this to data_fusion, or leave it as is?

Reply:
I think that, for now, let's make this an optional import. What I'd like to do in the long-run, though, is add the ability to have CLI plugins.

So, for now, would you please change this to be an optional import and add an issue to discuss whether CLI plugins are needed. Maybe they'd be useless after creating the order-based procflow, though, so just a discussion issue, not an actual "do this" issue.

Reply:
Sure! I can do that. I'll create an issue discussing your comment above as well.

Reply:
See #631.

Reply:
Thanks @evrose54